ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml

```diff
@@ -588,7 +591,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-release-4.5-e2e-gcp-upgrade
-    optional: true
```

Comment:
Dropping optional is surprising to me.  @abhinavdahiya , can you speak to the intended behavior of e2e-gcp-upgrade in 4.5?